src/index.js

```diff
@@ -29,6 +31,40 @@ export default function supernova(env) {
       const element = useElement();
       const rect = useRect();
       const Theme = useTheme();
+      const [linked, setLinked] = useState(false);
+      const selectionsAPI = useSelections();
+      useAction(
+        () => ({
+          action() {
+            setLinked(!linked);
+          },
+          icon: {
+            d: 'svg path',
+          },
+        }),
+        [linked]
+      );
+
+      useEffect(() => {
+        const addKeyPress = event => {
+          if (event.key === 'Shift') {
+            setLinked(true);
+          }
+        };
+
+        const removeKeyPress = event => {
+          if (event.key === 'Shift') {
+            setLinked(false);
+          }
+        };
+        document.addEventListener('keydown', addKeyPress);
+        document.addEventListener('keyup', removeKeyPress);
+
+        return () => {
+          document.removeEventListener('keydown', addKeyPress);
+          document.removeEventListener('keyup', removeKeyPress);
+        };
+      }, []);
```

Comment:
Since there is no button for chained selection yet I put this in place so we can test this. We might keep this in, but need to discuss with UX I guess.